2021/ahmed2022re/openreview/content.tex

```diff
@@ -0,0 +1,378 @@
+
+\paragraph{No. of models v/s Variability}
+In this experiment, we used the original settings, except for the number of trained models considered for the variability analyses. By changing the number of trained models for nondeterminism, we observed the change in model variability. Due to lack of time, we experiment with only 3 different sources of nondeterminism. We found that the result is not significantly different from the number of models, except for the error bars associated with standard deviation of accuracy and cross entropy. While all important metrics remain the same, it can be observed that these error bars decrease as the total number of models increases, as can be seen in Figure \ref{fig:pic1}. All sources tested in Table \ref{table:table8} show the same trend.
+ \begin{table}[!htb]
+\centering
+\resizebox{\columnwidth}{!}{
+	\begin{tabular}{c|c|c|c|c|c}
+	\hline
+    Setting & Accuracy & Cross-Entropy & Pairwise  & Pairwise & Ensemble \\
+            & SD \text{(\%)} & SD \text{(\%)} & Disagree \text{(\%)} & Corr. & $\Delta \text{(\%)}$\\
+	\hline
+		All Sources\text{/}(N=10) & $0.25 \pm 0.041$ & $0.0068 \pm 0.0010$  &  10.7 & 0.870 & 1.85  \\
+		All Sources\text{/}(N=25) & $0.26 \pm 0.034$ & $0.0073 \pm 0.0007$  &  10.7 & 0.873 & 1.86  \\
+	    All Sources\text{/}(N=50)&  $0.24 \pm 0.026$ & $0.0076 \pm 0.0007$  &  10.7 & 0.871 & 1.86   \\
+	    All Sources\text{/}(N=75)  & $0.25 \pm 0.021$ & $0.0078 \pm 0.0006$  &  10.7 & 0.872 & 1.87  \\
+	    All Sources\text{/}(N=100) & $0.26 \pm 0.018$ & $0.0080 \pm 0.0005$  &  10.7 & 0.872 & 1.87  \\
+	    \hline
+	    Data Shuffling\text{/}(N=10) & $0.25 \pm 0.042$ & $0.0061 \pm 0.0010$  &  10.8 & 0.871 & 1.86  \\
+	    Data Shuffling\text{/}(N=25) & $0.22 \pm 0.035$ & $0.0080 \pm 0.0009$ &  10.7 & 0.870 & 1.89  \\
+	    Data Shuffling\text{/}(N=50) & $0.25 \pm 0.027$ & $0.0079 \pm 0.0007$  &  10.7 & 0.870 & 1.88  \\
+	    Data Shuffling\text{/}(N=75) & $0.25 \pm 0.026$ & $0.0084 \pm 0.0006$  &  10.7 & 0.870 & 1.87  \\
+	    Data Shuffling\text{/}(N=100) & $0.25 \pm 0.021$ & $0.0082 \pm 0.0005$  &  10.7 & 0.871 & 1.86  \\
+	    \hline
+	    Random Bit Change\text{/}(N=10) & $0.26 \pm 0.047$ & $0.0063 \pm 0.0011$  &  10.7 & 0.874 & 1.88\\
+	    Random Bit Change\text{/}(N=25) & $0.23 \pm 0.027$ & $0.0075 \pm 0.0009$  &  10.6 & 0.874 & 1.85\\
+	    Random Bit Change\text{/}(N=50) & $0.22 \pm 0.019$ & $0.0069 \pm 0.0006$  &  10.6 & 0.874 & 1.83\\
+	    Random Bit Change\text{/}(N=75) & $0.22 \pm 0.017$ & $0.0070 \pm 0.0005$  &  10.6 & 0.874 & 1.83\\
+	    Random Bit Change\text{/}(N=100) & $0.21 \pm 0.014$ & $0.0068 \pm 0.0004$  & 10.6 & 0.874 & 1.82\\
+		\hline
+	\end{tabular}
+	}
+	\caption{No. of models v/s Variability. N denotes total number of trained models.}
+	\label{table:table8}
+\end{table}
+
+\begin{figure}[!htb]
+	\centering
+	\includegraphics[width=0.5\textwidth]{graph_1.png}
```

Comment:
This needs to be a relative import. Change to `../openreview/graph_1.png` to make it work.